### PR TITLE
Feature/iiif

### PR DESCRIPTION
**Pull request by jechols**
- Adds a check for pyramidal derivative and uses openseadragon if said derivative exists
- Adds config files for IIIF endpoint

### CODE REVIEW

**Comment by jechols** [comment]
Noteworthy stuff:

I added the IIP config because I'd spent the time figuring it out when I thought we could use that.  Figured we should keep it.

I also stuck with the awkward branching in the catalog show page more to get this up than as a permanent solution.  I think we'll probably be okay for now with something like this, but long-term we *will* need a better way to choose the viewer.

**Comment by terrellt** [comment]
Just to put some notes here rather than in slack: Need to point directly to the pyramidal directory, update NullDerivativePath to support #to_iiif, and make NullDerivativePath respond to #present? so we can trigger off that.

File: app/values/derivative_path.rb [hunk 1]
```diff
@@ -7,16 +7,38 @@ def initialize(string_or_path, path_factory=CapistranoAwarePath)
     super(path_factory.new(string_or_path))
   end
 
+  def present?
```
Comment (terrellt) [inline app/values/derivative_path.rb]: I don't think this is necessary, Rails hack

Comment (jechols) [inline app/values/derivative_path.rb]: You are correct!

**Pull request merged**